src/applications/claims-status/components/StemClaimListItemV3.jsx

```diff
@@ -37,7 +37,7 @@ export default function StemClaimListItemV3({ claim }) {
   return (
     <ClaimCard
       title="Edith Nourse Rogers STEM Scholarship application"
-      subtitle={`Submitted on ${formattedReceiptDate}`}
+      subtitle={`Received on ${formattedReceiptDate}`}
```

Comment:
This claim details card is used when the following is true:

claim type = 'education_benefits_claims'
useLighthouse = true
useNewClaimCards is disabled.